lib/capybara_accessible_selectors/filters/current.rb

```diff
@@ -1,7 +1,9 @@
 # frozen_string_literal: true
 
 Capybara::Selector::FilterSet[:capybara_accessible_selectors].instance_eval do
-  expression_filter(:current, skip_if: nil) do |xpath, value|
-    xpath[XPath.attr(:"aria-current") == value.to_s]
+  expression_filter(:current) do |xpath, value|
+    value = value.nil? ? false : value.to_s
+
+    builder(xpath).add_attribute_conditions("aria-current": value)
```

Comment:
Does stringifying this key in the hash assigner effect things at all?

I noticed in the codebase there are only symbol keys (Some wrapped in strings for some odd reason like here).

I've not looked at this codebase for 2+ years though. So ignore if I'm wrong

Reply:
Hmm oki, feels very odd but I guess I'm used to the clean standard assigner. This is required because of the non-standard nature of the key. So ignore my message. Used to snake-casing